src/libs/parsers/venueType.ts

```diff
@@ -0,0 +1,59 @@
+import { VenueTypeCode } from 'api/gen'
+import { IconInterface } from 'ui/svg/icons/types'
+import { Type } from 'ui/svg/icons/venueTypes'
+
+// Map the facetFilter (in algolia) to the label displayed in the front
+const MAP_TYPE_TO_LABEL: { [k in VenueTypeCode]: string } = {
+  ARTISTIC_COURSE: 'Cours et pratique artistiques',
+  BOOKSTORE: 'Bibliothèque ou médiathèque',
+  CONCERT_HALL: 'Musique - Salle de concerts',
+  CREATIVE_ARTS_STORE: 'Magasin arts créatifs',
+  CULTURAL_CENTRE: 'Centre culturel',
+  DIGITAL: 'Offre numérique',
+  FESTIVAL: 'Festival',
+  GAMES: 'Jeux / Jeux vidéos',
+  LIBRARY: 'Librairie',
+  MUSEUM: 'Musée',
+  MUSICAL_INSTRUMENT_STORE: 'Musique - Magasin d’instruments',
+  MOVIE: 'Cinéma - Salle de projections',
+  OTHER: 'Autre',
+  PATRIMONY_TOURISM: 'Patrimoine et tourisme',
+  PERFORMING_ARTS: 'Spectacle vivant',
+  RECORD_STORE: 'Musique - Disquaire',
+  SCIENTIFIC_CULTURE: 'Culture scientifique',
+  VISUAL_ARTS: 'Arts visuels, arts plastiques et galeries',
+}
+
+export const parseType = (types: VenueTypeCode | null | undefined): string => {
+  if (types && types in MAP_TYPE_TO_LABEL) return MAP_TYPE_TO_LABEL[types]
+  return MAP_TYPE_TO_LABEL.OTHER || ''
```

Comment:
Le typage n'indique pas que MAP_TYPE_TO_LABEL.OTHER puisse être `undefined`:
```suggestion
  return MAP_TYPE_TO_LABEL.OTHER
```